INSTALL.rst

```diff
@@ -99,6 +99,31 @@ Now visit the following URL in your browser:
 
         $ docker-machine ip <machine-name>
 
+Docker Dev Environment installation
+-----------------------------------
+
+It is possible to use a docker development environment, for testing purposes. You can create and run it
+by following the next steps:
+
+- Build your ui by using the command `yarn build`, after installing the dependencies. The ``index.html``
+  file should be inside the ``ui/cap-react/dist folder``, in oreder to be mounted and used by the containers.
+- Use the command `docker-compose -f docker-compose.dev.yml up` to build and start the containers.
+  It is probably also necessary to open another shell and use `docker-compose -f docker-compose.full.yml run web-api sh scripts/init.sh`.
+
+Now you have a dev environment that can automatically reload changed code in the backend, and will also
+accept changes in the frontend, after rebuilding the ``index.html`` file.
+
+If you want to debug your backend code, you will need to use the standard debugger, `import pdb; pdb.set_trace()`,
+and attach a new shell to the ``web-api`` container, like this: `docker attach cap_web-api_1`. The next time the debugger is triggered,
```

Comment:
* `pdb` or` ipdb` will work
* Same, please move `docker attach cap_web-api_1` to code block. Also, maybe we should replace `cap_web-api_1` with <CONTAINER_ID>` (and a small guide to get the container_id, e.g. docker ps, and get container id of `cap-web-api` or of required container)